docs/input-kafka.asciidoc

```diff
@@ -133,6 +150,7 @@ See the https://kafka.apache.org/{kafka_client_doc}/documentation for more detai
 | <<plugins-{type}s-{plugin}-sasl_client_callback_handler_class>> |<<string,string>>|No
 | <<plugins-{type}s-{plugin}-sasl_oauthbearer_token_endpoint_url>> |<<string,string>>|No
 | <<plugins-{type}s-{plugin}-sasl_oauthbearer_scope_claim_name>> |<<string,string>>|No
+| <<plugins-{type}s-{plugin}-sasl_iam_jar_paths>> |<<array,array>>|No
```

Comment:
not ordered alphabetically :D

Reply:
`sasl_iam_jar_paths` should stay before `sasl_login_callback_handler_class`, or we also fix `sasl_oauthbearer*` `sasl_jaas_config` and `sasl_kerberos_service_name` ordering. Maybe sorting the whole settings section should be done in another PR.